packages/algob/src/lib/algo-operator.ts

```diff
@@ -302,10 +302,10 @@ export class AlgoOperatorImpl implements AlgoOperator {
     };
   }
 
-  async OptInToSSC (sender: Account, index: number, payFlags: TxParams): Promise<void> {
+  async OptInToSSC (sender: Account, appId: number, payFlags: TxParams): Promise<void> {
```

Comment:
1. The method name should be lowercase
2. Please add a docstring comment explaning arguments (especially `appId`)

Reply:
ok